app/assets/stylesheets/buttons.scss

```diff
@@ -8,4 +8,7 @@
   -webkit-appearance: none;
   font-stretch: condensed;
   font-weight: bold;
+  &.cta-danger {
+    background: $red;
+  }
```

Comment:
Uhh, I can't wait to kill that file entirely but we still have bunch of legacy buttons sooo.. 🤷‍♂️

Reply:
We can probably replace all of the legacy buttons in a separate PR :)